docs/FileInput.md

```diff
@@ -152,6 +153,10 @@ If that's not enough, you can pass a `placeholder` prop to overwrite it. The val
 </FileInput>
 ```
 
+## `removeIcon`
+
+Optionally overwrite the [default icon](https://mui.com/material-ui/material-icons/?query=removeCir&selected=RemoveCircle) for removing files.
```

Comment:
Could you add a small code example showing how one can use `removeIcon` to change the default icon to something else? (take inspiration from the other sections of the doc :wink:)

Reply:
Done, and I tried to keep it consistent with the other sections.